trollflow2/plugins/__init__.py

```diff
@@ -40,6 +40,10 @@
 from pyresample.boundary import AreaDefBoundary, Boundary
 from pyresample.area_config import AreaNotFound
 from rasterio.enums import Resampling
+try:
+    from trollmoves.movers import S3Mover
+except ImportError:
+    S3Mover = None
```

Comment:
Could we make the s3 uploader plugin a separate module so that we can just import this without the try except?

Reply:
I moved the import inside the plugin in https://github.com/pytroll/trollflow2/pull/158/commits/f45e03e0d7175d349bf6703ea303db1c32c0b655 so the try/except isn't needed.